content/post/azure-monitor-terraform.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Azure Monitor & Terraform: Collecting custom log files"
+date: 2023-10-03T14:46:23+01:00
+hero: /images/hero-3.jpg
```

Comment:
I see the following picture is one of the parts from a split image. https://github.com/rakutentech/engineering-blog/blob/main/static/images/hero-3.jpg

make sure you are using the correct one